server/container/guice/mailet/pom.xml

```diff
@@ -94,6 +94,11 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.james.jdkim</groupId>
+            <artifactId>apache-jdkim-library</artifactId>
```

Comment:
Why do we need that dependency?

Reply:
`PublicKeyRecordRetriever`, `MultiplexingPublicKeyRecordRetriever` and `DNSPublicKeyRecordRetriever`

Reply:
Well, doesn't dkim mailet have a transitive dependency?